psl/diagnostics/src/warning.rs

```diff
@@ -20,13 +21,38 @@ impl DatamodelWarning {
         DatamodelWarning { message, span }
     }
 
-    pub fn new_feature_deprecated(feature: &str, span: Span) -> DatamodelWarning {
+    pub fn new_preview_feature_deprecated(feature: &str, span: Span) -> DatamodelWarning {
         let message = format!(
             "Preview feature \"{feature}\" is deprecated. The functionality can be used without specifying it as a preview feature."
         );
         Self::new(message, span)
     }
 
+    pub fn new_preview_feature_renamed(
+        deprecated_feature: &str,
+        renamed_feature: impl Display,
+        prisly_link_endpoint: &str,
+        span: Span,
+    ) -> DatamodelWarning {
+        let message = format!(
+                "Preview feature \"{deprecated_feature}\" has been renamed as \"{renamed_feature}\". Learn more at https://pris.ly/d/{prisly_link_endpoint}."
```

Comment:
```suggestion
                "Preview feature \"{deprecated_feature}\" has been renamed to \"{renamed_feature}\". Learn more at https://pris.ly/d/{prisly_link_endpoint}."
```

Reply:
I've used `renamed as` since this form was already used [elsewhere](https://github.com/prisma/prisma-engines/blob/f8f78f335fd86dea323d7fbc581fdf500d745e9a/schema-engine/sql-introspection-tests/tests/re_introspection/relation_mode/mysql.rs#L77) (in a comment, not a publicly visible diagnostic), and [it seems to be correct in English](https://textranch.com/c/renamed-to-or-renamed-as/#:~:text=Both%20%22renamed%20to%22%20and%20%22renamed%20as%22%20are%20correct,suggesting%20a%20transformation%20or%20reclassification%20of%20the%20entity.).

^ that is just for context, I'm fine with using `renamed to` as you suggest.